test/test_authn.py

```diff
@@ -90,11 +88,9 @@ def test_authn_signin_wrong_credentials(client):
 
 def test_authn_signout(client):
     resp = client.post(URL_SIGNIN, json=CREDENTIALS_0)
-    assert resp.json["code"] == "ok"
 
     resp = client.get(URL_SIGNOUT)
-    assert resp.status_code == 302
-    assert resp.headers["Location"] == "/login"
+    assert resp.status_code == 200
```

Comment:
Let's check the message, no?

```suggestion
    assert resp.status_code == 200
    assert resp.json["msg"] == ""
```

~Even better if we had a meaningful message instead of empty string~:

```suggestion
    assert resp.status_code == 200
    assert resp.json["msg"] == "Successful sign out"
```